README.md

```diff
@@ -1,83 +1,87 @@
+Concerning bugs please use the [issue tracker](https://github.com/thsmi/sieve/issues) or send a private email to `schmid-thomas at gmx.net`. You find more details on reporting bugs in the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md)
 
 
-Give me 1-2 weeks time for a reply. If you did not receive a reply at all, it
-might be a good idea to check your spam filter.
+Please allow 1-2 weeks for a reply. If you did not receive a reply at all, it might be a good idea to check your spam filter.
+
 
 ## License
 
-The extension is free and open source software, it is made available to you
-under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
+The [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) is free and open source software, it is made available to you under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
 
 Refer to [Licensing information](https://github.com/thsmi/sieve/blob/master/LICENSING_INFO.md) for details.
 
+
 ## Releases
 
-You are looking for the most recent release?
+You are looking for the most recent [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) release?
 
-Just go to [the releases page and download the latest stable xpi or view the release notes](https://github.com/thsmi/sieve/releases/latest).
+[Available by downloading the latest stable xpi at the Releases page (release notes can also be viewed)](https://github.com/thsmi/sieve/releases/latest).
 
-**Firefox user have to do a right click on the XPI download and select "Save link as" otherwise Firefox will steal the link and try to install the addon it into Firefox which will fail**
+**Firefox users: At the [ATN Sieve add-on page](https://addons.thunderbird.net/addon/sieve/), right-click on the “+ Download Now” button and select “Save Link As…”; otherwise Firefox try to install the add-on into Firefox which will fail.**
 
-The addon is restartless, so there is no need to restart Thunderbird upon installation and deinstallation.
+The add-on is restartless, so there is no need to restart Thunderbird upon installation and uninstallation.
 
-**Sometimes Thunderbird fails to invalidate its cache when updating a restartless addon. The result is the new version is displayed in the addons dialog but the old cached code is still be used. In such a case uninstall the addon, then restart thunderbird, restart it again and finally install it again. This ensures the cache gets correctly invalidated.** 
+**Sometimes Thunderbird fails to validate its cache when updating a restartless add-on. The result is the new version is displayed in the add-ons dialog but the old cached code is still used. In such cases, uninstall the add-on, then restart Thunderbird, restart it again and finally install it again. This ensures the cache gets correctly validated.** 

```

Comment:
Validate would change the meaning of the sentence. Technically the add-on validation succeeds in thunderbird, but it fails when [marking the caches as invalid, which is invalidate](https://www.merriam-webster.com/dictionary/invalidate) and the opposite of validate.

Reply:
Will change as you requested.